servicetalk-grpc-api/src/main/java/io/servicetalk/grpc/api/GrpcClientMetadata.java

```diff
@@ -42,4 +43,15 @@
      * <a href="https://www.grpc.io">gRPC</a> method.
      */
     ContentCodec requestEncoding();
+
+    /**
+     * Deadline after which the client no longer wants response.
+     *
+     * @return {@link Duration} of associated deadline or null if no deadline. All durations greater than 99999999 hours
```

Comment:
Consider using `{@code ...}` for `null` and `99999999`

Reply:
Yes for the constant, no for null. Stephen Colebourne's JSR310 Time style Javadoc has become somewhat of defacto standard and it doesn't use `{@code }` for null, true and false. https://blog.joda.org/2012/11/javadoc-coding-standards.html